bin/tessel-2.js

```diff
@@ -516,6 +518,38 @@ makeCommand('root')
 
 
 module.exports = function(args) {
+  var sIndexOfSA = -1;
+  var eIndexOfSA = -1;
+
+  // Check to see if there are any subargs...
+  for (var i = 0; i < args.length; i++) {
+    var arg = args[i];
+
+    if (arg.startsWith('[') && sIndexOfSA === -1) {
+      // Remove the leading '[', replace existing
+      args[i] = arg.slice(1, arg.length);
+      sIndexOfSA = i;
+    }
+
+    if (arg.endsWith(']') && sIndexOfSA !== -1) {
+      // Remove the trailing ']'
+      args[i] = arg.slice(0, arg.length - 1);
+      eIndexOfSA = i;
+    }
+  }
```

Comment:
It would've been nice to use `subarg` to parse this stuff, but in reality we don't actually want to parse these yet because there is no clear path to reassembling them as the string they will need to be when the remote process is invoked.